pkg/k8shandler/forwarding.go

```diff
@@ -0,0 +1,223 @@
+package k8shandler
+
+import (
+	"fmt"
+
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/util/sets"
+
+	logging "github.com/openshift/cluster-logging-operator/pkg/apis/logging/v1"
+	logforward "github.com/openshift/cluster-logging-operator/pkg/apis/logging/v1alpha1"
+	"github.com/openshift/cluster-logging-operator/pkg/generators/forwarding"
+	"github.com/openshift/cluster-logging-operator/pkg/logger"
+	"github.com/openshift/cluster-logging-operator/pkg/utils"
+)
+
+const (
+	internalOutputName       = "clo-default-output-es"
+	collectorSecretName      = "fluentd"
+	logStoreService          = "elasticsearch.openshift-logging.svc:9200"
+	defaultAppPipelineName   = "clo-default-app-pipeline"
+	defaultInfraPipelineName = "clo-default-infra-pipeline"
+	secureForwardConfHash    = "8163d9a59a20ada8ab58c2535a3a4924"
+
+	//ForwardingAnnotation  Annotate CL instance with a value of "enabled"
+	ForwardingAnnotation = "clusterlogging.openshift.io/logforwarding"
```

Comment:
this is the critical annotation to enable the TP feature, right?  Can we put TP in the name?

e.g. `clusterlogging.openshift.io/logforwardingtechpreview`